styles.css

```diff
@@ -3,69 +3,186 @@
 }
 
 body {
-    font-family: 'Barlow', sans-serif;
-    padding: 2rem;
+    font-family: 'Rubik', sans-serif;
     color: #262729;
+    display: grid;
+    grid-gap: 10px;
+    grid-template-rows: 1fr auto 0.2fr;
+    margin: 0; 
+    background-color: #ecdfdb;
+    line-height: 1.5;
 }
 
 header {
-    background-color: moccasin;
+    background-color: #3b27ba;
+    text-align: center;
+    padding-bottom: 1.5rem;
+    color: #13ca91;
 }
 
+header a {
+    text-decoration: none;
+    color: #13ca91;
+  }
+
 .nav {
-    color: 
+    padding: 1rem;
 }
 
-
 h1 {
-    color: indianred;
+    font-style: bold;
+    font-weight: 800;
 }
 
 h2 {
     text-decoration: underline;
+    color: #3b27ba;
 }
 
-
-table {
-    background-color: gainsboro;
-    border: solid black 1px; 
-    width: 800px;
-    font-weight: 100;
+h3 {
+    color: #3b27ba;
 }
 
-th {
-    text-align: left;
-    padding: 1rem 1rem;
+.linkcontainer {
+    display: flex;
+    justify-content: space-around;
+    border: solid 0.1rem #13ca91;
 }
 
-th.date {
-    width: 10%;
+.linkcontainer_item {
+    padding: 1rem;
+    text-align: center;
 }
 
-th.job {
-    width: 30%
+.hobbies, .skills {
+    text-align: center;
+    border: solid 0.1rem #13ca91;
 }
 
-th.company {
-    width: 50%;
+ul {
+    padding: 0;
 }
 
-.linkcontainer {
-    display: flex;
-    justify-content: center;
+li {
+    display: inline-block;  
+    background-color: #13ca91;
+    padding: 0.4rem;  
 }
 
-.linkcontainer_item {
-    border: solid 1px black;
+.aboutme {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 0.05fr 0.5fr auto 0.3fr;
+    grid-gap: 0.5rem;
     padding: 1rem;
+}
+
+.headline {
+    grid-column: 1 / 3;
+    grid-row: 1 / 2;
+    font-weight: bold;
+}
+
+.contact {
+    grid-column: 2 / 3;
+}
+
+.portrait {
+    grid-column: 1 / 2;
     text-align: center;
-    width: 300px;
 }
 
+.biography {
+    grid-column: 1 / 3;
+}
 
+.links {
+    grid-column: -3 / -1;
+    padding: 0 1rem;
+}
 
+.cv {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    padding: 1rem;
+}
+
+.date {
+    color: grey;
+}
+
+.image {
+    width: 70%;
+    padding: 1rem 0 0 0; 
+}
+
+.icon {
+    width: 20px;
+}
+
+.headline-cv {
+    grid-column: 1 / -1;
+}
+
+.professional-experience {
+    padding: 0 1rem;
+    grid-columns: 1 / 2;
+}
 
+.studies {
+    padding: 0 1rem ;
+}
+
+.footer {
+    background-color: #3b27ba;
+    color: #13ca91;
+    padding: 1rem;
+}
 
+footer p img {
+    text-align: right;
+}
 
+a {
+    color: #3b27ba;
+    text-decoration: none;
+}
 
+.stations {
+    border-left: solid 0.2rem #13ca91;
+    padding: 0 0.5rem;
+}
+
+.contact-item {
```

Comment:
Try adding a flexbox to your contact items. I think you can then easily center your icons and text.